src/decorators/__tests__/is-boolean.decorator.test.ts

```diff
@@ -36,7 +35,7 @@ describe('decorators', () => {
 
             const property = getProperty(Example, 'foo');
 
-            expect(property?.decorator).toEqual(DecoratorType.IsBoolean);
+            expect(property?.decorator).toEqual('IsBoolean');
```

Comment:
I chose to eliminate the enum because we now use a string union type. We could have kept the enum but I don't think it's adding much any more. And `tsc` ensures that we only use valid strings for these keys.